### ISSUE TITLE
Multi-language switching

### ISSUE THREAD

**Issue by tmtmtmtm** [mysociety/popolo-viewer-sinatra#151]
If we have data in multiple languages, provide a toggle for switching between them.

**Comment by tmtmtmtm** [comment]
@zarino, this isn't implemented yet, but it would be handy to have the design-elements part of it ready to drop in when it is. I'm imagining this as something similar to the house-switcher, but very open to any suggestions!

**Comment by zarino** [comment]
@tmtmtmtm: Would it change the language for the whole page, or just the names of members, parties and areas in the tables at the bottom?

**Comment by tmtmtmtm** [comment]
@zarino This would solely be about data, rather than any surrounding interface. So, it would for example change "Republican Party of Labour and Justice" into "Respublikanskaja Partija Truda i Spravedlivosti" — not only throughout the data table, but also the seat count groupings at the top, etc — but the column would still be called "Party", rather than "Partija", and it would still say "8 seats" etc.

**Comment by zarino** [comment]
Ok. So it'd translate the "entities" on the page, like party names, but nothing we generate ourselves.

**Comment by tmtmtmtm** [comment]
Yes, both sets can theoretically vary separately. You might want both an English and Spanish language view of data that exists in Arabic and French. This is only about the data half — translating the interface is something for much later.
